### DISCUSSION TITLE
Will there be a disruptive renaming event in the near future?

### DISCUSSION THREAD

**Discussion by sheldonhull** [aquaproj/aqua#1319] [Deprecated (Question and Answer)] [answered]
I'm starting to rely on this tool a bit more, including with github actions. 

Is there still a goal to rename the project? If so it could be impactful to leveraging aqua if the actual repo and other core names are updated. 

Will there be a simple upgrade path?

I definitely am advocating for more usage of this project, but this is a concern I want to catch up front.

**Comment by suzuki-shunsuke** [comment, marked as answer]
- https://github.com/aquaproj/aqua/issues/881

We discussed the renaming of this project but currently the issue is inactive.
So we don't have any plan to rename this project in the near future at the moment.

The name `aqua` is bad in terms of searchability so I'd like to improve the searchability,
but I couldn't rename the project because I don't have a good idea of naming and the renaming would cause the breaking changes.

**Reply by sheldonhull** [reply]
It is tricky for searching I agree, but to be honest I'm not sure that `aqua-vm` is more clear. The only thing I can think of that would help is a fundamentally different name (like `binbrew` or something that's unique) and that would be a breaking change. 

I'm trying to leverage this ecosystem as it's so much better than `asdf` and `brew` for the development type binaries in my opinion, but it's not so visible yet. I'm not confident that is due to the name, or more the lack of awareness of how much simpler this is to get up and running with.

There's a few things that are a "negative" for those not used to using renovate style based updates, in that it's a lot more work to keep updated without the tooling.
While the current toolchain is amazing, I think that a compromise for `aqua upgrade` and `aqua update` that prompt for upgrade/registry updates would go a long way towards making things more accessible. Is it as good as renovate and PR tool updates... _no_. However, in dev tooling some level of simplicity could be a major plus. 

Additionally, using for github actions (I'm doing this right now as a test case) is tricky because the config doesn't allow splitting or tagging the app (I'll submit a discussion idea for this). 

I'll do my best to keep promoting! Really nice work and I hope to see a lot more collaborators on this!
